xAPI-Data.md

```diff
@@ -405,7 +406,7 @@ The Actor defines who performed the action. The Actor of a Statement can be an A
 
 <a name="agent"/>
 
-##### <a name="2.4.2.1">2.4.2.1</a> When the Actor ObjectType is Agent
+##### <a name="2.4.2.1">2.4.2.1</a> When the Actor ObjectType is `Agent`
```

Comment:
I maybe got a bit zealous on these.  I did ignore other formatting in titles, so I'm going to back off on these.